Editor/UI/Wizard/FirstTime/XRSDKSetupPage.cs

```diff
@@ -15,31 +15,44 @@ internal class XRSDKSetupPage : WizardPage
     {
         private enum XRLoader
         {
-            None,
```

Comment:
Not important, but usually I try to keep a value that does not affect further logic first so new instances don't get preconfigured immediately.